profiles/arch/riscv/package.use.mask

```diff
@@ -1,6 +1,14 @@
 # Copyright 2019-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Ye Cao <ye.c@rioslab.org> (2021-06-30)
+# # Dependencies not keyworded, not tested
+sci-libs/fftw mpi
+media-sound/alsa-utils libsamplerate
+dev-python/ipython doc matplotlib nbconvert notebook
+dev-python/matplotlib cairo excel tk wxwidgets 
```

Comment:
Don't mask _tk_, the necessary dependencies got keyworded yesterday.

Reply:
I think I could unmask this in another commit after this PR?  It's quite hard for me to update my local branch to the latest changes. Solving this seperately after merging will be much easier.

Reply:
Thank you! fixed it.